### DISCUSSION TITLE
Why my page:start in defineNuxtPlugin doesn't work

### DISCUSSION THREAD

**Discussion by xie392** [nuxt/nuxt#15993] [Questions]
I created a plugin for loading when the page jumps. But I don't know why page:start doesn't work

![image](https://user-images.githubusercontent.com/61953437/211039216-685171d4-f0da-4ab4-94ba-d62eda31df13.png)

![image](https://user-images.githubusercontent.com/61953437/211038386-f7118b26-8291-4f44-a4bc-aecbea66399d.png)

nuxt version: 3.0.0
System: win10

**Comment by kanopa** [comment]
page:start is called when navigating to another page. you should just use the component from the nuxt team which also supports custom loader display https://nuxt.com/docs/api/components/nuxt-loading-indicator#nuxtloadingindicator

**Comment by xie392** [comment]
I seem to have found the problem, but don't know how to fix it. I have two layouts, the loading progress bar will only take effect when using default.vue, and it will fail when I enable account.vue

![image](https://user-images.githubusercontent.com/61953437/211130061-3bf0dbde-2175-4ece-9121-3a80dc534a7a.png)

![image](https://user-images.githubusercontent.com/61953437/211130092-49d42450-8873-4c00-a46e-24c6992adadc.png)

![image](https://user-images.githubusercontent.com/61953437/211130143-7d1dbeb7-7275-4d8e-baa6-fc39992cec62.png)
